examples/website/src/shared/route/blockAdapters.ts

```diff
@@ -90,8 +90,14 @@ function getBlocks(blocks: any, articleMeta?: ArticleMeta): Block[] {
           key: index,
           value: {
             text: block.text,
+            richText: block.richText,
             linkURL: block.linkURL,
-            linkText: block.linkText
+            linkText: block.linkText,
+            linkTarget: block.linkTarget,
+            styleOption: block.styleOption,
+            layoutOption: block.layoutOption,
+            imageID: block.image?.id,
+            image: block.image && imageAdapter(block.image)
```

Comment:
again why both `image` and `imageID`?

Reply:
its from before, i did not updated all examples, corrected

Reply:
Will you change this?

Reply:
yes

Reply:
![image](https://user-images.githubusercontent.com/15183757/98789314-6bad8300-2402-11eb-9f82-5ad46df72280.png)

Its changed, it seems to be an GH UI bug, i dunno ... see here https://github.com/bajour/wepublish/blob/f/page-break-extend/examples/website/src/shared/route/blockAdapters.ts#L100